test/serializers/associations_test.rb

```diff
@@ -44,7 +44,7 @@ def test_has_many
         assert_equal(
           { posts: { type: :has_many, options: { embed: :ids } },
             roles: { type: :has_many, options: { embed: :ids } },
-            bio: { type: :belongs_to, options: {} } },
```

Comment:
Am I misunderstanding, or is this not removing the the test for the `belongs_to`?